### PR TITLE
[AccHack] Fix tab navigation for the Project Gallery tabs

### PR DESCRIPTION
**Pull request by Erin007**
https://codedotorg.atlassian.net/browse/A11Y-63

Using tab navigation, the Project Gallery tabs are now accessible. 

https://github.com/code-dot-org/code-dot-org/assets/12300669/b5be520e-85f6-4d4d-b266-bbe18c6c4241

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/projects/ProjectsGallery.jsx [hunk 1]
```diff
@@ -51,11 +51,19 @@ class ProjectsGallery extends Component {
...
             <div
               key={tab.key}
+              tabIndex="0"
+              role="button"
               style={[
                 styles.pill,
                 this.props.selectedGallery === tab.key && styles.selectedPill,
               ]}
               onClick={() => this.toggleTo(tab)}
+              onKeyDown={e => {
```
Comment (hannahbergam) [inline apps/src/templates/projects/ProjectsGallery.jsx:61]: For my own learning- can you help me understand how the different keys work here? Is this a way for 'tab' to move through, and 'spacebar' or 'enter' to effectively 'select'?

Comment (Erin007) [reply apps/src/templates/projects/ProjectsGallery.jsx:61]: You got it! `onKeyDown` is just an event handler that let's you specify what should happen when certain keys are pressed. In this case, when space and enter are pressed, we basically want to "click" on that tab to navigate to the selected tab's content.

Comment (hannahbergam) [reply apps/src/templates/projects/ProjectsGallery.jsx:61]: Excellent. I should've just looked into the definition of 'toggleTo' to confirm!

**Review by hannahbergam — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**